src/builders/dev-server/index.ts

```diff
@@ -3,38 +3,52 @@ import { BuilderContext } from '@angular-devkit/architect';
 import { virtualFs, Path, tags } from '@angular-devkit/core';
 import { SingleSpaBuilderSchema } from '../browser/schema';
 import { Configuration } from 'webpack';
+import { SingleSpaWebpackBuilder } from '../single-spa-webpack-builder';
 import * as fs from 'fs';
 import * as path from 'path';
-import { SingleSpaBrowserBuilder } from '../browser';
 import * as webpackMerge from 'webpack-merge';
 
 // @ts-ignore
 export class SingleSpaDevServer extends DevServerBuilder {
     constructor(context: BuilderContext) {
         super(context);
-    }   
+    }
 
     buildWebpackConfig(
         root: Path,
         projectRoot: Path,
         host: virtualFs.Host<fs.Stats>,
         options: SingleSpaBuilderSchema
     ): Configuration {
-        const browserBuilder = new SingleSpaBrowserBuilder(this.context);
-        const webpackConfig = browserBuilder.buildWebpackConfig(root, projectRoot, host, options);
-        this.context.logger.info(tags.oneLine `
-        SINGLE-SPA-ANGULAR: Angular dev server is serving application as a single module`);
+        // Disable es2015 polyfills
+        // tslint:disable-next-line: max-line-length
+        // https://github.com/angular/angular-cli/blob/3d8064bb64d72557474a7484f1b85eaf35788916/packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts#L56
+        options.es5BrowserSupport = false;
+        // Generate Angular CLI's default builder webpack cofiguration
+        const config = super.buildWebpackConfig(root, projectRoot, host, options);
+
+        // Delegate the building of the webpack config to the new builder.
+        // Builder based on custom builders implmented by @meltedspark
+        // https://github.com/meltedspark/angular-builders
+        const webpackConfig = SingleSpaWebpackBuilder.buildWebpackConfig(root, options.singleSpaWebpackConfigPath, config, options, this.context);
+        this.context.logger.info(tags.oneLine`
+        [single-spa-angular]: Angular dev server is serving application as a single module`);
         return webpackConfig;
-    }    
+    }
     _buildServerConfig(root, projectRoot, options, browserOptions) {
         // @ts-ignore
         const devServerConfig = super._buildServerConfig(root, projectRoot, options, browserOptions);
+        const contentBase = path.resolve(root, projectRoot.serveDirectory || '../');
```

Comment:
This causes bugs in Windows. We should not use `path.resolve()` for this, since webpack needs paths that have `/` instead of `\`. See related release notes for beta 7, 8, and 9 at https://github.com/CanopyTax/single-spa-angular/releases

Reply:
Not a problem. Just as an FYI though, this has been working fine on my Windows 10 Insider Preview Skip-Ahead build.